### DISCUSSION TITLE
Tags at line level

### DISCUSSION THREAD

**Discussion by latvia234** [vincode-io/Zavala#114] [Ideas]
At this time [tags are only available at the outline level](https://github.com/vincode-io/Zavala/discussions/56#discussioncomment-1136747).

I am curious if there are more people who want tags at line level. I think that tags at line level are very useful.

It would be great to have the option to choose between searching for a tag in one outline, or in all outlines. (Maybe how Dynalist handles tags?)

**Comment by vincode-io** [comment]
I've created issue #122 to implement this. I'm not sure when it will be implemented, but it for sure won't be before the 2.0 release which is way down the line.
